src/System.Text.Utf8/src/System/Text/Utf8/Utf8String.CodePointReverseEnumerator.cs

```diff
@@ -11,40 +11,14 @@ partial struct Utf8String
         // TODO: Name TBD
         public struct CodePointReverseEnumerator : IEnumerator<UnicodeCodePoint>, IEnumerator
         {
-            // TODO: This object is heavier than string itself... Once we got ByteSpan runtime support, change it.
-            // TODO: Reduce number of members when we get Span<byte> runtime support
-            private ByteSpan _startBuffer;
-            private ByteSpan _resetBufferPosition;
-            private ByteSpan _buffer;
-
-            private byte[] _bytes;
-            private int _startIndex;
+            private Span<byte> _buffer;
             private int _index;
-            private int _endIndex;
-            private const int ResetIndex = -UnicodeConstants.Utf8MaxCodeUnitsPerCodePoint - 1;
-
             private int _currentLenCache;
+            private const int ResetIndex = -UnicodeConstants.Utf8MaxCodeUnitsPerCodePoint - 1;
```

Comment:
I agree although I'm not good with naming - let's not introduce refactoring in this simple conversion review. I wanted to spend separate time on the style.